orm/query.go

```diff
@@ -1484,6 +1500,9 @@ func (q *Query) appendReturning(fmter QueryFormatter, b []byte) (_ []byte, err e
 func (q *Query) appendWith(fmter QueryFormatter, b []byte) (_ []byte, err error) {
 	b = append(b, "WITH "...)
 	for i, with := range q.with {
+		if with.recursive {
+			b = append(b, "RECURSIVE "...)
```

Comment:
I think you should add something to check whether or not a space needs to be appended here. If `i` is greater than 0 then this will result in `WITH RECURSIVE , ` rather than `WITH RECURSIVE, `. It's not a huge deal but it makes the generated query look a lot better when debugging.